### PR TITLE
PR #03: Test Cases for Scenarios 01

### PR DESCRIPTION
**Pull request by harveyvn**
In this Pull Request, I added some test cases for Scenario 01:
- test_EGO_and_NPC_on_different_lane
- test_EGO_and_NPC_on_same_lane
- test_EGO_behind_NPC_on_same_lane
- test_EGO_between_TRUCK_and_SEDAN_on_same_lane
- test_EGO_on_different_land_and_ego_between_SUV_and_SEDAN
- test_EGO_changes_lane

There are some samples added to check the working status of Simulation Connection. Please execute the `runner.py` and then you will be able to verify the result. The screencast is also attached below.

runner.py

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -327,7 +325,6 @@ celerybeat.pid
 
 # Environments
 .env
-.venv
```
Comment (TrackerSB) [inline .gitignore:330]: Why did you remove the VirtualEnv folder?

Comment (harveyvn) [reply .gitignore:330]: I removed those items because they are duplicated. For example, .venv is already defined on line 139.

Comment (TrackerSB) [reply .gitignore:330]: I see. 👍🏻

**Review by TrackerSB — APPROVED** [review]
